shared/macros.jinja

```diff
@@ -20,6 +20,9 @@ This rule's check operates on the cluster configuration dump.
 Therefore, you need to use a tool that can query the OCP API, retrieve the {{% if endpoint is string %}}<code class="ocp-api-endpoint">{{{ endpoint }}}</code> API endpoint to the local <code class="ocp-dump-location">{{{ xccdf_value("ocp_data_root") }}}/{{{ endpoint.lstrip("/") }}}</code> file.{{% else %}}{{% for item in endpoint %}}<code class="ocp-api-endpoint">{{{ item }}}</code> API endpoint to the local <code class="ocp-dump-location">{{{ xccdf_value("ocp_data_root") }}}/{{{ item.lstrip("/") }}}</code> file{{% endfor %}}.{{% endif %}}
 {{%- endmacro %}}
 
+{{% macro openshift_resource_path_xccdf(varname) -%}}
+Customized to: <code class="ocp-api-endpoint-var">{{{ varname }}}</code>
```

Comment:
It would be good to add more verbiage here; this appears on the security guide and the report and it would be good to still give the appropriate context to people.

Reply:
I changed code to use the text substitution functionality discussed in https://github.com/openshift/compliance-operator/pull/750 and it avoids me from using a new macro `openshift_resource_path_xccdf`.  Instead, the code uses an existing macro `openshift_cluster_setting`.

Reply:
Hi @tmishina, just looked at the PR, I am wondering is a reason why are we using `openshift_resource_path_xccdf`, instead of openshift_cluster_setting("/apis/apps/v1/namespaces/{{<somvar>}}"

Reply:
@Vincent056 sorry, my operation mistake.  I've just force-pushed the latest code which does not use `openshift_resource_path_xccdf`.

Reply:
No worries, just let me know when you have the updated code.

Reply:
@Vincent056 Now the branch contains the updated code.  `macros.jinja` does not include `openshift_resource_path_xccdf`, and no rule uses it. Sorry for confusion.